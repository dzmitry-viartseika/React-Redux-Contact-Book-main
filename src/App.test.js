import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import App from './App';

configure({adapter: new Adapter()});

test('test', () => {
    const component = shallow(<App />);
    const wrapper = component.find('.App');
    expect(wrapper.length).toBe(1);
});