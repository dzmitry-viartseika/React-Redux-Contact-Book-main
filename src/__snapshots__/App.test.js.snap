// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly App component 1`] = `
<div
  className="App"
>
  <ToastContainer
    autoClose={5000}
    closeButton={[Function]}
    closeOnClick={true}
    draggable={true}
    draggableDirection="x"
    draggablePercent={80}
    hideProgressBar={false}
    newestOnTop={false}
    pauseOnFocusLoss={true}
    pauseOnHover={true}
    position="top-right"
    role="alert"
    rtl={false}
    transition={[Function]}
  />
  <Navbar />
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/add"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/edit/:id"
  />
</div>
`;

exports[`unit tests for App component 1`] = `
<div
  className="App"
>
  <ToastContainer
    autoClose={5000}
    closeButton={[Function]}
    closeOnClick={true}
    draggable={true}
    draggableDirection="x"
    draggablePercent={80}
    hideProgressBar={false}
    newestOnTop={false}
    pauseOnFocusLoss={true}
    pauseOnHover={true}
    position="top-right"
    role="alert"
    rtl={false}
    transition={[Function]}
  />
  <Navbar />
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/add"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/edit/:id"
  />
</div>
`;
